pkg/trace/stats/concentrator.go

```diff
@@ -55,37 +58,24 @@ type Concentrator struct {
 	statsd                 statsd.ClientInterface
 }
 
-var defaultPeerTags = []string{
-	"_dd.base_service",
-	"amqp.destination",
-	"amqp.exchange",
-	"amqp.queue",
-	"aws.queue.name",
-	"bucketname",
-	"cassandra.cluster",
-	"db.cassandra.contact.points",
-	"db.couchbase.seed.nodes",
-	"db.hostname",
-	"db.instance",
-	"db.name",
-	"db.system",
-	"hazelcast.instance",
-	"messaging.kafka.bootstrap.servers",
-	"mongodb.db",
-	"msmq.queue.path",
-	"net.peer.name",
-	"network.destination.name",
-	"peer.hostname",
-	"peer.service",
-	"queuename",
-	"rpc.service",
-	"rulename",
-	"server.address",
-	"statemachinename",
-	"streamname",
-	"tablename",
-	"topicname",
-}
+//go:embed peer_tags.ini
+var peerTagFile []byte
+
+var defaultPeerTags = func() []string {
+	cfg, _ := ini.Load(peerTagFile)
```

Comment:
We need to check for an error here and provide some sort of more gentle failure. Otherwise if there's any error loading or parsing the file we'll just panic with a `nil` pointer error.